JourneysSquad/JourneysSquad/Helpers/OutputPreparator.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+/// Praparator for data that shown un the `BillRowView`.
+struct OutputPreparator {
+    // MARK: - Internal interface
+
+    /// Creates a string by joining the elements of an array with commas and spaces.
+    ///
+    /// - Parameter input: The array of strings to be joined.
+    /// - Returns: A string containing the array elements separated by commas and spaces.
+    static func makeCommaSeparatedList(_ input: [String]) -> String {
+        input.joined(separator: ", ")
+    }
+
+    /// Adds a currency symbol to a decimal number and returns the formatted string representation.
+    ///
+    /// - Parameter number: The decimal number to which the currency symbol should be added.
+    /// - Returns: A string representation of the decimal number with the currency symbol.
+    static func addCurrencyKind(to number: Decimal) -> String {
+        number.formatted(.currency(code: Self.currencyCode))
+    }
```

Comment:
It can be something like:

```suggestion
/// Returns a string representation of the given number formatted as amount of money
    /// in the specified currency.
    ///
    /// - Parameters:
    ///   - number: A decimal number to represent as amount of money.
    ///   - currency: A currency code of the currency to represent.
    /// - Returns: A string representing the given number as an amount of money in the specified currency.
    static func formattedAsAmountOfMoney(_ number: Decimal, code: String) -> String {
        number.formatted(.currency(code: code))
    }
```